README.md

```diff
@@ -15,6 +15,16 @@ The following channels are set up:
 - PUT  <http://localhost:5001/patient-registration/{FHIR_ID}>
 - POST <http://localhost:5001/immunization>
 
+### Patient Registration/Update flow
```

Comment:
@de-laz it looks good man. I think we should move the section up so that it comes before the examples and also change the heading to h2. We should also add a link to the docasaurus page

Reply:
@bradsawadye It sounds like a good idea. Thanks :+1: